client/src/components/TagField.js

```diff
@@ -12,19 +13,24 @@ class TagField extends Component {
     };
 
     this.onChange = this.onChange.bind(this);
+    this.handleOnBlur = this.handleOnBlur.bind(this);
     this.getOptions = this.getOptions.bind(this);
   }
 
   onChange(value) {
     this.setState({
-      value
+      value: value
     });
 
     if (typeof this.props.onChange === 'function') {
       this.props.onChange(value);
     }
   }
 
+  handleOnBlur (e) {
+
+  }
```

Comment:
What's the purpose of this?

Reply:
Hey man, see my comments on the react slack channel. Was a bug with react-select https://github.com/JedWatson/react-select/issues/805

Reply:
Ok sweet, can you add a docblock with that issue link in it maybe?